certbot/CHANGELOG.md

```diff
@@ -2,7 +2,23 @@
 
 Certbot adheres to [Semantic Versioning](https://semver.org/).
 
-## 2.11.1 - main
+## 2.11.2 - master
```

Comment:
in no way accurate but also whatever

Reply:
:shrug: the script wanted a next version